tonic/models/vic/grid_params.py

```diff
@@ -322,100 +569,323 @@ def __init__(self, organic_fract=False, spatial_frost=False,
                           'root_depth': 'Root zone thickness (sum of depths is'
                                         ' total depth of root penetration)',
                           'root_fract': 'Fraction of root in the current root '
-                                        'zone.',
-                          'LAI': 'Leaf Area Index, one per month'}
+                                        'zone'}
+        if blowing_snow:
+            self.veg_param['sigma_slope'] = 'Std. deviation of terrain ' \
+                                            + 'slope within veg tile'
+            self.veg_param['lag_one'] = 'Lag one gradient autocorrelation ' \
+                                        + 'of terrain slope'
```

Comment:
the backslash and plus sign are not needed here. Python can do the string line continuation as:

```Python
a = 'start of the '
    'line'

Reply:
Not only does my editor complain, but when I attempt to run tonic in my python environment, python complains about unexpected indent on the continued lines.  Restoring the \ and + appeases my python.  I'm running Python 3.5.

Reply:
@tbohn - My apologies, my comment was not entirely correct. You don't need the `+` or `\` if you are inside parentheses. When you are not (as in this case), you need the backslash but not the plus sign. 

ref: https://stackoverflow.com/questions/5437619/python-style-line-continuation-with-strings